src/app/components/list/overview/index.html

```diff
@@ -32,7 +32,7 @@
       defaultValue="SkyListInMemoryDataProvider"
       isOptional="true"
     >
-    Specifies a data provider to obtain the data to display. The list component requires this property or the <sky-code>data</sky-code> property. Acceptable values: <sky-code>extends ListDataProvider</sky-code>. To extend ListDataProvider, provide a <sky-code>get</sky-code> function that accepts a <sky-code>ListDataRequestModel</sky-code> as an argument and returns an <sky-code>Observable ListDataResponseModel </sky-code>, and a <sky-code>count</sky-code> function that returns an <sky-code>Observable number</sky-code> that indicates how many total records are in the list. For lists that use <sky-code>dataProvider</sky-code> instead of <sky-code>data</sky-code>, consumers are responsible for managing all <sky-code>ListDataRequestModel</sky-code> properties. For examples of custom searching, filters, and paging, see the <a stacheRouterLink="." fragment="code">demo code</a> below.
+    Specifies a data provider to obtain the data to display. The list component requires this property or the <sky-code>data</sky-code> property. Acceptable values: <sky-code>extends ListDataProvider</sky-code>. To extend ListDataProvider, provide a <sky-code>get</sky-code> function that accepts a <sky-code>ListDataRequestModel</sky-code> as an argument and returns an <sky-code>Observable ListDataResponseModel </sky-code>, and a <sky-code>count</sky-code> function that returns an <sky-code>Observable number</sky-code> that indicates how many total records are in the list. For lists that use <sky-code>dataProvider</sky-code> instead of <sky-code>data</sky-code>, consumers are responsible for managing <a stacheRouterLink="." fragment="listdatarequestmodel-properties">all <sky-code>ListDataRequestModel</sky-code> properties</a>. For examples of custom searching, filters, and paging, see the <a stacheRouterLink="." fragment="code">demo code</a> below.
```

Comment:
Superminor, but can you please delete the extra space at the end of the `Observable ListDataResponseModel` code snippet?